php/Terminus/Commands/SiteCommand.php

```diff
@@ -2142,25 +2130,19 @@ public function upstreamUpdates($args, $assoc_args) {
             isset($assoc_args['accept-upstream'])
             && $assoc_args['accept-upstream']
           );
-          $message  = 'Are you sure you want to apply the ';
-          $message .= 'upstream updates to %s-dev';
+          $message        = 'Are you sure you want to apply the ';
+          $message       .= 'upstream updates to %s-dev';
           $this->input()->confirm(
             array(
               'message' => $message,
               'context' => array($site->get('name'), $env),
             )
           );
-          $workflow = $site->applyUpstreamUpdates(
-            $env,
-            $updatedb,
-            $acceptupstream
-          );
+          $workflow = $site->applyUpstreamUpdates($env, $updatedb, $acceptupstream);
```

Comment:
I'm not sure which of our PRs ought to go first - mine changes the way you access upstreams, but yours fixes a bug. Would you be amenable to splitting the fix to applyUpstreamUpdates into another PR (with a CHANGELOG entry about the fix) please, @ronan?